schemas/past_officers.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ocf.berkeley.edu/schemas/past_officers.schema.json",
+  "type": "object",
+  "properties": {
+    "terms": {
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/term"
+      }
+    }
+  },
+  "required": [
+    "terms"
+  ],
+  "$defs": {
+    "term": {
+      "type": "object",
+      "required": [
+        "termSemester",
+        "termOfficers"
+      ],
+      "properties": {
+        "termSemester": {
+          "type": "string"
+        },
+        "termOfficers": {
+          "type": "object",
+          "required": [
+            "gms",
+            "sms"
+          ],
+          "properties": {
+            "gms":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "sms":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "gmsDeputies":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "smsDeputies":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "committeeHeads":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            }
+          }
+        }
+      }
+    },
+    "pastOfficer": {
+      "type": "object",
+      "required": [
+        "officerHandle",
+        "officerFirstName",
+        "officerLastName"
+      ],
+      "properties": {
+        "officerHandle": {
+          "type": "string"
+        },
+        "officerFirstName": {
```

Comment:
Don't have separate "first name" and "last name" fields. It's best to have one "full name" field.

(I don't have a good source for this piece of advice, so you'll have to trust me that it's best practice.)

Reply:
I think since people's names can be sometimes don't fit just the first/last mold (from Dutch tussenvoegsels to multiple last names) generally just using a full name is best.